doc/Language/containers.pod6

```diff
@@ -166,7 +166,7 @@ bind (whether C<=> or C<:=> is used):
 =head1 Scalar containers and listy things
 
 There are a number of positional container types with slightly different
-semantics in Perl 6. The most basic one is L<List|/type/List>;
+semantics in Perl 6. The most basic one is L<List|/type/List>, which
 it is created by the comma operator.
```

Comment:
Asking as a non-native person, is "which it is created" correct here over "which is created"?

Reply:
@Altai-man you're right, of course. Fixed https://github.com/perl6/doc/commit/6995071999cb4385b7a060df10d253e68f3a0a83